### DISCUSSION TITLE
on Login 500 Error is coming

### DISCUSSION THREAD

**Discussion by AnandKumar019** [community/community#33122] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello,

I have been trying to login into my GitHub Account but it saying 500 error.
Does any body else is having this issue and how can I resolve it?

**Comment by tamalCodes** [comment]
It's a server issue from their side, 
Other pages are working fine except for `github.com` , it'll be fixed ! 

You can close this discussion, and mark my answer !

**Comment by radar-blip** [comment, marked as answer]
For account related issues: try here: [Support page](https://support.github.com/contact/account)

**Reply by AnandKumar019** [reply]
Thanks for the Help @linuxlawson :-)

**Reply by radar-blip** [reply]
You're welcome. Good luck!! 👍🏻

**Reply by tamalCodes** [reply]
> For account related issues: try here: [Support page](https://support.github.com/contact/account)

That's not account related issues since alot of folks have the same issue. All the other pages are working fine, spamming support won't help 😂

**Comment by srjchauhan** [comment]
i am facing the same issue.

**Reply by tamalCodes** [reply]
Same here, hopefully they fix it soon !

**Comment by coolgovind** [comment]
I am also facing the same issue

**Reply by tamalCodes** [reply]
Same here, hopefully they fix it soon !

**Reply by coolgovind** [reply]
Yes hoping the same

**Comment by manojgajare123** [comment]
I am also facing the same issue

**Comment by radar-blip** [comment]
Same thing happening to several people right now. Signing into your profile page seems to work fine.

`https://github.com/username`

**Reply by manojgajare123** [reply]
No it is not fine it is show 500 error

**Reply by jayashankar987** [reply]
issues seems to be in `github.com` url, all other `github.com/*` urls are working

**Comment by ZinkLu** [comment]
same issue, home page 500.

**Comment by aswad98** [comment]
same issue,home page 500

**Comment by tamalCodes** [comment]
It's fixed.

**Reply by radar-blip** [reply]
Yep!

**Reply by liquidsmok3** [reply]
I cant log new issues and get error 500 still.
